frontend/javascripts/oxalis/view/context_menu.js

```diff
@@ -444,6 +444,14 @@ function ContextMenu(props: Props) {
 
   return (
     <React.Fragment>
+      <div
+        className="node-context-menu-overlay"
+        onClick={hideContextMenu}
+        onContextMenu={evt => {
+          evt.preventDefault();
+          hideContextMenu();
+        }}
+      />
       <div
         style={{
           position: "absolute",
```

Comment:
```suggestion
```
This property can be removed as it is already included in the css.

Reply:
good catch!

Reply:
Sorry my bad,
this was not redundant and just introduced a bug.
I am opening a quick fix pr now